java_build.sh

```diff
@@ -17,4 +17,4 @@ protoc --java_out=${OUTDIR} opentelemetry/proto/resource/v1/resource.proto \
 OPENAPI_OUTDIR="./build/gen-openapi"
 
 mkdir -p $OPENAPI_OUTDIR
-protoc --plugin=protoc-gen-swagger=/usr/bin/protoc-gen-swagger --swagger_out=logtostderr=true,grpc_api_configuration=opentelemetry/proto/agent/trace/v1/trace_service_http.yaml:$OPENAPI_OUTDIR opentelemetry/proto/agent/trace/v1/trace_service.proto
+#protoc --plugin=protoc-gen-swagger=/usr/bin/protoc-gen-swagger --swagger_out=logtostderr=true,grpc_api_configuration=opentelemetry/proto/agent/trace/v1/trace_service_http.yaml:$OPENAPI_OUTDIR opentelemetry/proto/agent/trace/v1/trace_service.proto
```

Comment:
curious about this change...

Reply:
Ahh, slipped when committing, reverting.

Reply:
Fixed.